src/js/components/Menu/Menu.js

```diff
@@ -70,7 +70,11 @@ const Menu = forwardRef((props, ref) => {
   } = props;
   const theme = useContext(ThemeContext) || defaultProps.theme;
   const iconColor = normalizeColor(theme.menu.icons.color || 'control', theme);
-  const align = dropProps.align || dropAlign;
+  // need to destructure the align otherwise it will get passed through
+  // to DropButton and override prop values
+  const { align: menuAlign, ...themeDropProps } = theme.menu.drop;
```

Comment:
Can we rename "menuAlign" to "themeDropAlign"? It feels a bit more descriptively accurate.

Reply:
Yep! Will adjust now.

Reply:
Done.